Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/GGVPointer.cs

```diff
@@ -190,7 +190,15 @@ public void OnPreSceneQuery()
         public virtual Vector3 Position => sourcePosition;
 
         /// <inheritdoc />
-        public virtual Quaternion Rotation => Quaternion.LookRotation(gazeProvider.GazePointer.Rays[0].Direction);
+        public virtual Quaternion Rotation
+        {
+            get
+            {
+                Vector3 look = Position - CameraCache.Main.transform.position;
+                look.y = 0;
```

Comment:
Nit: Can we also add a comment about the intent here (i.e. otherwise it's future someone asking why Y specifically is being zeroed out)